mix/MixClient.cpp

```diff
@@ -180,7 +180,7 @@ void MixClient::executeTransaction(Transaction const& _t, State& _state, bool _c
 		else
 			levels.resize(ext.depth);
 
-		machineStates.emplace_back(MachineState({steps, vm.curPC(), inst, newMemSize, vm.gas(),
+		machineStates.emplace_back(MachineState({steps, vm.curPC(), inst, newMemSize, (u256)gas,
```

Comment:
Maybe break this into each argument on its own line, according to the coding standards?